bedrock/base/tests/test_helpers.py

```diff
@@ -45,6 +45,46 @@ def test_mailtoencode_with_unicode(self):
         context = {"key": "\xe4"}
         assert render(template, context) == '<a href="?var=%C3%A4">'
 
+    def test_add_bedrock_attributes(self):
+        fresh_html = """
+            <html>
+            <body>
+                <h1>Test h1</h1>
+                <p>No change</p>
+                <h2>Test h2 one</h2>
+                <ul>
+                    <li>one</li>
+                </ul>
+                <h2>Test h2 two</h2>
+                <ol>
+                    <li>two</li>
+                </ol>
+                <h3>Test @#$% h3</h3>
+                <p>Here is an <a href="https://example.com">external link</a></p>
+            </body>
+        </html>
+        """
+        expected_html = """
```

Comment:
This is fine/workable as is, but you mentioned in Slack how it made you a bit 🤢 -- so you may find Python's [`textwrap.dedent`](https://docs.python.org/3/library/textwrap.html#textwrap.dedent) to be handy, but don't feel like you have to shoehorn that in right now

Reply:
It did not immediately work so I'll leave it for a future improvement.